tests/Tests/400__ods_dashboard/413__ods_dashboard_settings_pod_tolerations.robot

```diff
@@ -22,11 +22,10 @@ Verify Toleration Is Applied To Pod
     ...     ODS-1685
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook With Arguments    image=s2i-minimal-notebook
-    Verify Pod Toleration    TestToleration
+    Verify Server Pod Has The Expected Toleration    TestToleration
 
 
 *** Keywords ***
-
 Set Pod Toleration Via UI
```

Comment:
Could be renamed to `Set Notebook Pod Toleration`

Reply:
I think this name is fine